README.md

```diff
@@ -1,25 +1,28 @@
-[![Build CI](https://github.com/tonyketcham/p5-svelte/actions/workflows/build.yml/badge.svg)](https://github.com/tonyketcham/p5-svelte/actions/workflows/build.yml)
-[![Netlify Status](https://api.netlify.com/api/v1/badges/7dd33ce7-b638-4540-83d1-4442ef026efc/deploy-status)](https://app.netlify.com/sites/p5-svelte/deploys)
-[![npm version](https://badge.fury.io/js/p5-svelte.svg)](https://badge.fury.io/js/p5-svelte)
+<br>
 
-<h1 align="center">🧙‍♂️ p5-Svelte 🕸
 <div align="center">
-<img src="https://res.cloudinary.com/practicaldev/image/fetch/s--5L4QI4B_--/c_limit%2Cf_auto%2Cfl_progressive%2Cq_66%2Cw_880/https://dev-to-uploads.s3.amazonaws.com/i/ozuet5qpew6t9rup6o4p.gif" alt="10print made using p5-svelte" width="265" height="265" />
+<img src="/static/logo.svg" alt="p5-svelte logo" width="80" />
```

Comment:
The logo link should be an absolute path URL so that it displays outside Github on npm, etc

Reply:
Oh, indeed, I didn't think about it that way, there you go.
Plus, it can't be the one in my repo, it's a bit counter-intuitive to link to a not-yet-existing file, you wouldn't be able to see the logo until the PR is merged.

Reply:
Good point, I'll follow up after merging in